CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+Amy is an open source project, and we welcome contributions of all
+kinds: new lessons, fixes to existing material, bug reports, and
+reviews of proposed changes are all equally welcome.
+
+By contributing, you are agreeing that Software Carpentry may
+redistribute your work under [this license][license].  You also agree
+to abide by our [contributor code of conduct][conduct].
+
+If you would like to know what we need help with, please see the
+[issues][issues] on GitHub.
```

Comment:
Where the anchor name matches the link text, you can leave the anchor name blank.  So `[issues][]`.